gradle.properties

```diff
@@ -3,5 +3,4 @@ org.gradle.configuration-cache=true
 org.gradle.jvmargs=-Dfile.encoding=UTF-8 -Xmx4g
 org.gradle.parallel=true
 
-kotlin.stdlib.default.dependency=false
```

Comment:
Seems we still have to fix this warning:

```
> Task :plugin:verifyPluginConfiguration
[gradle-intellij-plugin :plugin plugin:plugin:verifyPluginConfiguration] The following plugin configuration issues were found:
- The dependency on the Kotlin Standard Library (stdlib) is automatically added when using the Gradle Kotlin plugin and may conflict with the version provided with the IntelliJ Platform, see: https://jb.gg/intellij-platform-kotlin-stdlib
See: https://jb.gg/intellij-platform-versions
```

https://plugins.jetbrains.com/docs/intellij/using-kotlin.html#kotlin-standard-library

Reply:
All warnings have been fixed.

https://github.com/ComposeGears/Valkyrie/actions/runs/10037674525/job/27737963189